tests/vm_test.go

```diff
@@ -871,16 +869,23 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				vmi, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
 
-				By("Triggering a segfault in qemu")
-				domSpec, err := libdomain.GetRunningVMIDomainSpec(vmi)
-				Expect(err).ToNot(HaveOccurred())
-				emulator := filepath.Base(domSpec.Devices.Emulator)
-				libpod.RunCommandOnVmiPod(vmi, []string{"killall", "-11", emulator})
+				vmiPod, err := libpod.GetPodByVirtualMachineInstance(vmi, vmi.Namespace)
+				Expect(err).NotTo(HaveOccurred())
 
-				By("Ensuring the VM stops")
-				Eventually(ThisVM(vm), 360*time.Second, 1*time.Second).Should(HavePrintableStatus(v1.VirtualMachineStatusStopped))
+				By("Killing the VirtualMachineInstance")
+				Expect(pkillVMI(kubevirt.Client(), vmi)).To(Succeed(), "Should deploy helper pod to kill VMI")
+				Eventually(ThisPod(vmiPod), 30*time.Second, 1*time.Second).Should(BeInPhase(v1.Failed))
 
 				By("Waiting for VM to start again")
+				Eventually(func() k8sv1.PodPhase {
```

Comment:
please use KubeVirt APIs for this (VM, VMI), not an implementation detail (virt-launcher pod). The VM user should be able to know that its guest crashed.